### DISCUSSION TITLE
夏日归家小记 | /posts/summer-go-home2/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#628] [Announcements]
https://yuanfan.rbind.io/posts/summer-go-home2/

**Comment by yihui** [comment]
国字去掉外面三画不就是玉么，或者去掉里面的横横点就剩下田。

正好我的丝瓜今天也是开了第一朵花。

香瓜和烧瓜弄错不奇怪啊，因为很可能是前一年自留的种子。虽然瓜很不一样，但瓜种子都很像，第二年自己也想不起来留的是哪种瓜了，种下去长出来之后才能知道。

**Reply by earfanfan** [reply]
哇，禾几大神的脑筋好会转弯呀。既然可以得到田，那么阡陌继续延伸也是可以的，比如甲由申。话又说回来，我爷爷的答案是目。我跟堂妹依依也是只想到了玉。

难道二十四节气在你家那边也是应验的么。我还搜了一下，公历的一年365天是地球绕太阳公转一圈，而农历的一年是再结合了月相，二十四节气是按地球绕太阳转的轨道等分24份。这么说起来大暑节气前后，是一年最热的几天，地球在公转轨道上离太阳最近，地球自转一圈，就是正好地球两端的中国和美国都很热的时候，于是乎，我爷爷种的丝瓜开花了，你种的丝瓜也开花了。哈哈哈，这充分说明大家都生活在一个地球上。

香瓜种错成烧瓜的原因，确实就是留的种子错了，这点只有你这样的老农民才能体会呐。
